docker-compose.yaml

```diff
@@ -0,0 +1,32 @@
+services:
+  cinema_service:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+      - ./:/cinema_service
```

Comment:
Mounting the entire project directory can lead to permission issues and include unnecessary files. Consider refining the mounted directories to include only what's necessary for the application to run.